src/Command/ImportReferentNominationCommand.php

```diff
@@ -62,37 +86,21 @@ protected function initialize(InputInterface $input, OutputInterface $output)
     {
         $this->em = $this->getContainer()->get('doctrine.orm.entity_manager');
         $this->referentAreaRepository = $this->em->getRepository(ReferentArea::class);
+        $this->referentRepository = $this->em->getRepository(Referent::class);
         $this->mediaRepository = $this->em->getRepository(Media::class);
-        $this->mediasInfo = [
-            [
-                'path' => 'avatar_femme_01.jpg',
-                'name' => 'Avatar femme 1',
-                'sex' => 'F',
-            ],
-            [
-                'path' => 'avatar_femme_02.jpg',
-                'name' => 'Avatar femme 2',
-                'sex' => 'F',
-            ],
-            [
-                'path' => 'avatar_homme_01.jpg',
-                'name' => 'Avatar homme 1',
-                'sex' => 'M',
-            ],
-            [
-                'path' => 'avatar_homme_02.jpg',
-                'name' => 'Avatar homme 2',
-                'sex' => 'M',
-            ],
-        ];
         $this->mediaFactory = $this->getContainer()->get('app.content.media_factory');
         $this->storage = $this->getContainer()->get('app.storage');
     }
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $rows = file($input->getArgument('fileUrl'));
-        if ($rows === false) {
+        $rows = [];
+        if (false !== ($handle = fopen($input->getArgument('fileUrl'), 'r'))) {
```

Comment:
if (false === ($handle = fopen($input->getArgument('fileUrl'), 'r')))) {
    $output->writeln('Error : File Not Found');

    return 1;
}

/// csv parsing